sdk/trace/provider.go

```diff
@@ -141,7 +142,7 @@ func (p *TracerProvider) UnregisterSpanProcessor(s SpanProcessor) {
 	}
 	if stopOnce != nil {
 		stopOnce.state.Do(func() {
-			s.Shutdown()
+			_ = s.Shutdown(context.Background())
```

Comment:
```suggestion
			global.Handle(s.Shutdown(context.Background()))
```

Should we handle this error instead of dropping it?

Reply:
Sure, done